src/test/java/api/loans/DueDateNotRealTimeScheduledNoticesProcessingTests.java

```diff
@@ -133,10 +134,11 @@ public void beforeRecurringNoticesAreRescheduled()
     NoticePolicyBuilder noticePolicy = new NoticePolicyBuilder()
       .withName("Policy with due date notices")
       .withLoanNotices(Collections.singletonList(uponAtDueDateNoticeConfig));
-    useLoanPolicyAsFallback(
+    useFallbackPolicies(
```

Comment:
Can you replace this by `use(noticePolicy)`?

Reply:
Done.